scripts/split_repo_migration/split-repo-post-process.sh

```diff
@@ -0,0 +1,260 @@
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+NOP="echo -n"
+DEBUG="yes"  # set to blank for a real run
+MESSAGE=""
+
+[[ -n ${DEBUG} ]] && {
+  # Comment out some commands for development/debugging
+  RM='echo "*** would run:" rm'
+  GIT='echo "*** would run:" git'
+
+  # It's easier to debug if keys don't get reordered
+  SORT_JSON_KEYS="" 
+} || {
+  # Real commands for actual runs
+  RM='rm'
+  GIT='git'
+
+  # Sort keys for easier look-up.
+  SORT_JSON_KEYS="--sort-keys"
+}
+
+if [ $# -lt 1 ]
+then
+  echo "Usage: $0 PATH/TO/PACKAGE_NAME"
+  exit 1
+fi
+
+# variable naming convention
+#   PATH_* are absolute system paths
+#   MONOREPO_PATH_* are paths relative to the root of the monorepo
+#   MONOREPO_* are values used in the monorepo
+
+# In a real run for a split repo already copied to the mono-repo, $1 would be
+# specified as `packages/PACKAGE_NAME`. For developing this script, it's useful
+# to specify $1 as the name of a completely separate split repo, in which case
+# specifying $2 and $3 is also useful (see below).
+PATH_PACKAGE="$(realpath "$1")"
+
+# The optional second argument is useful for development: it specifies a different
+# directory for the monorepo (rather than deriving it from $1).
+PATH_MONOREPO="${2:-$(dirname $(dirname $PATH_PACKAGE))}"
+PATH_MONOREPO="$(realpath "${PATH_MONOREPO}")"
+
+# The optional third argument is useful for development: it specifies what the the
+# package name should be for this API once migrated to the monorepo. This can be
+# inferred if we've already migrated the split repo to the correct location, but
+# otherwise needs to be specified, since it differs from the split repo name
+# (for example, the repo "python-speech" defines a package
+# "google-cloud-speech").
+MONOREPO_PACKAGE_NAME="${3:-$(basename ${PATH_PACKAGE})}"
+
+MONOREPO_PATH_PACKAGE="packages/${MONOREPO_PACKAGE_NAME}"
+
+cat <<EOF
+Post-processing ${MONOREPO_PACKAGE_NAME}
+  PATH_PACKAGE:          ${PATH_PACKAGE}
+  PATH_MONOREPO:         ${PATH_MONOREPO}
+  MONOREPO_PACKAGE_NAME: ${MONOREPO_PACKAGE_NAME}
+  MONOREPO_PATH_PACKAGE: ${MONOREPO_PATH_PACKAGE}
+EOF
+
+pushd "${PATH_MONOREPO}" >& /dev/null
+
+## START system tests check ########################################
+# variable prefix: TST_*
+
+# If there are integration tests, do not proceed with the script.
+
+TST_MONO_TESTDIR="${MONOREPO_PATH_PACKAGE}/tests/"
+TST_MONO_SYSTEM_DIR="${TST_MONO_TESTDIR}system"
+TST_MONO_SYSTEM_FILE="${TST_MONO_TESTDIR}system.py"
+echo "Checking for system tests in ${TST_MONO_TESTDIR}"
+
+[[ ! -f ${TST_MONO_SYSTEM_FILE} ]] || \
+  { echo "ERROR: ${TST_MONO_SYSTEM_FILE} exists. Need to manually deal with that." ; return -10 ; }
+[[ ! -d ${TST_MONO_SYSTEM_DIR} ]] || \
+  { echo "ERROR: ${TST_MONO_SYSTEM_DIR} exists. Need to manually deal with that." ; return -11 ; }
+## END system tests check
+
+## START release-please config migration ########################################
```

Comment:
We will tackle the release-please configuration migration separately. I'm going to add do not merge while we determine the best way to maintain the release-please config file for all packages in the monorepo. I noticed issues with the config file in https://github.com/googleapis/google-cloud-python/pull/11051 when I tried to add the configuration for the workstations v1 version. See the changes required here:
https://github.com/googleapis/google-cloud-python/pull/11051/commits/ec374bee70b71757f3f6a2f85cc6db438e5cb7f5

Reply:
Sounds good.